vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -4,183 +4,201 @@ sidebarDepth: 2
+   * Click `Next`
+6. For `Node Group network configuration`, the subnets should already be setup and selected
+7. Select `Configure SSH access to nodes`. Follow the links to create a new SSH key pair if you don't already have one.
+8. Select `All` to allow all source IPs
+9. Click `Next`   
+10. Review your settings and then click `Create`. It may take a few minutes for the Node Group to be created.
+
+### Connect to the Cluster
+   1. *Note:* If this is a brand new setup you will need to configure the AWS CLI using the user account from the steps above. You'll need to provide your Access Key ID, Secret Key, and Region.
+```sh      
+aws configure
+```
+   2. Setup your Kubernetes context 
+```sh
+aws eks --region REGION-CODE update-kubeconfig --name YOUR-CLUSTER-1
+```
+   For example: `aws eks --region us-east-2 update-kubeconfig --name cluster-1`. More details and troubleshooting can be found here. <https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html>
+
+   3. Your current context should now be configured for your AWS cluster. Run the command below to check
+```sh
+kubectl config current-context
+```
+   Your output should look something like this: `arn:aws:eks:us-east-2:483173223614:cluster/cluster-1`
+ 
 ### Install the NGINX Ingress Controller
-1. Add the NGINX controller for ingress. This depends on your role having permissions for ELB.
-   - For basic NGINX ingress install run this command
-    ```sh
-    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.0.2/deploy/static/provider/cloud/deploy.yaml
-    ```
-   - See <https://kubernetes.github.io/ingress-nginx/deploy/#aws> as well as [this](https://docs.nginx.com/nginx/deployment-guides/amazon-web-services/ingress-controller-elastic-kubernetes-services/) for more detailed install steps.
+Add the NGINX controller for the ingress. This step relies on your role having permissions for Elastic Load Balancing (ELB).
+
+1. Create the NGINX ingress controller
+```sh
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/controller-v1.1.3/deploy/static/provider/aws/deploy.yaml
+```
 2. Get the ELB external URL for your NGINX install
-   - Run: ```kubectl get services -n ingress-nginx```
-   - Get the value of the external address (EXTERNAL-IP) for the ingress-nginx-controller:
+```sh
+kubectl get services -n ingress-nginx
+```
+
+For example:
 ```
 NAME                                 TYPE           CLUSTER-IP       EXTERNAL-IP                        
 ingress-nginx-controller             LoadBalancer   10.100.102.83    ad234bd11a1ff4dadb44639a6bbf707e-0e0a483d966405ee.elb.us-east-2.amazonaws.com
 ```
+The value of the external URL (EXTERNAL-IP) should be available within a few minutes. You'll use this address for `YOUR-HOST-NAME` in the steps below.
```

Comment:
if we're backticking placeholders then -> `EXTERNAL-IP`

Reply:
Per our meeting, no backticks on placeholders so I'll remove them.